ptvsd/wrapper.py

```diff
@@ -2222,6 +2231,43 @@ def on_exceptionInfo(self, request, args):
                      'source': source},
         )
 
+    @async_handler
+    def on_completions(self, request, args):
+        type_look_up = {
```

Comment:
This should be outside of the function, so that the dict is not re-created for every new request.